src/main/java/org/jabref/gui/ClipBoardManager.java

```diff
@@ -30,34 +37,56 @@
 import org.slf4j.LoggerFactory;
 
 public class ClipBoardManager {
```

Comment:
A better approach for singleton is to use an enum https://stackoverflow.com/a/26285591/3450689

Reply:
Source of the information: [Effective Java Item 3](https://www.oreilly.com/library/view/effective-java-3rd/9780134686097/). A highly-recommended reading. A summary of the book can be found at https://github.com/HugoMatilla/Effective-JAVA-Summary. (Just commented at stackoverflow - thank you for the pointer)

Reply:
Thanks, @Siedlerchr!